client/components/jetpack/card/jetpack-product-card/index.tsx

```diff
@@ -140,17 +140,17 @@ const DisplayPrice = ( {
 	);
 	const discountElt =
 		billingTerm === TERM_ANNUALLY
-			? translate( '* Save %(percent)d%% for the first year', {
+			? translate( 'Save %(percent)d%% for the first year ✢', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.